docs/pages/indexer.mdx

```diff
@@ -47,14 +47,191 @@ They are written under the assumption you are using `anvil` for your test chain,
+
+   If you are running [the minimal template](/tutorials/minimal), the expected output should be similar to:
+
+   ```
+   qbzzt1@penguin:~/clnt2$ pnpm indexerClient
+
+   > client@0.0.0 indexerClient /home/qbzzt1/clnt2
+   > npx ts-node --esm index.ts
+
+   Block number: 2042
+   Tables: schema,Hooks,StoreMetadata,NamespaceOwner,InstalledModules,ResourceAccess,Systems,FunctionSelector,SystemHooks,SystemRegistry,ResourceType,Counter
+   Information about Counter
+   {
+     id: '0x5FbDB2315678afecb367f032d93F642f64180aa3____Counter',
+     address: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+     tableId: '0x00000000000000000000000000000000436f756e746572000000000000000000',
+     namespace: '',
+     name: 'Counter',
+     keySchema: {},
+     valueSchema: { value: 'uint32' },
+     lastUpdatedBlockNumber: 2042n,
+     records: [ { key: {}, value: [Object] } ]
+   }
+   The actual value: 2
+   ```
+
+<details>
+<summary>Explanation</summary>
+
+```ts
+import { createIndexerClient } from "@latticexyz/store-sync/trpc-indexer";
+const indexer = createIndexerClient({
+  url: "http://localhost:3001",
+});
+```
+
+Create an indexer client. The URL, `http://localhost/3001`, is the one for the indexer you run on the local computer.
+If the indexer is elsewhere, modify the URL as appropriate.
+
+```ts
+const result = await indexer.findAll.query({
+  chainId: 31337,
+  address: "0x5FbDB2315678afecb367f032d93F642f64180aa3",
+});
+```
+
+Connect to the indexer.
+The `chainId` value is for a sanity check, to ensure we are connecting to the correct indexer.
+The `address` parameter is the address of the `World` contract whose information we are reading.
+The same indexer can read from multiple `World` objects, as long as they are on the same blockchain.
+
+```ts
+const blockNum = result.blockNumber;
+console.log(`Block number: ${blockNum}`);
+```
+
+The `result` has two fields:
+
+- `blockNumber`, the data is correct as of that block (it could change later).
+- `tables`, the tables of the `World` we are reading.
+
+```ts
+console.log(`Tables: ${result.tables.map((t) => t.name)}`);
+```
+
+Use [`map`](https://www.tutorialspoint.com/typescript/typescript_array_map.htm) to get the names of the tables.
+
+```ts
+const counterTable = result.tables.filter((t) => t.name == "Counter")[0];
+```
+
+Use [`filter`](https://www.tutorialspoint.com/typescript/typescript_array_filter.htm) to get the `Counter` table, assuming there is at least one.
+
+```ts
+console.log("Information about Counter");
+console.log(counterTable);
+```
+
+Log the information we have about the table.
+It contains these fields:
+
+| Field                  | Meaning                                                                                       |
+| ---------------------- | --------------------------------------------------------------------------------------------- |
+| id                     | The table's ID, which encodes the `World` address, the name space, and the name of the table. |
```

Comment:
this is just an internal ID for uniqueness in the context of SQL, doesn't correspond to anything on chain or in the protocol